content/en/references/configuration.md

```diff
@@ -64,16 +67,7 @@ Docker is used extensively by LocalStack, and there are several configuration pa
 
 ## Local AWS Services
 
-This section covers configuration values that are specific to certain AWS services.
-
-* [AppSync]({{< ref "#appsync" >}})
-* [Batch]({{< ref "#batch" >}})
-* [DynamoDB]({{< ref "#dynamodb" >}})
-* [Elastic Kubernetes Service (EKS)]({{< ref "#eks" >}})
-* [Elasticsearch]({{< ref "#elasticsearch" >}})
-* [Kinesis]({{< ref "#kinesis" >}})
-* [Lambda]({{< ref "#lambda" >}})
-* [Stepfunctions]({{< ref "#stepfunctions" >}})
```

Comment:
Removed because this was hopelessly out of date, and besides, there is a autogenerated index on the right